### PR TITLE
Wrapper Stacks Explanation

### PR DESCRIPTION
**Pull request by rcowper**
Included wrapper stacks explanation in main readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +13,9 @@ The StartupKit-templates repo contains a collection of AWS [CloudFormation](http
 - An [ElastiCache Cluster](https://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/WhatIs.html) ([jump](#amazon-elasticache-cluster))
 - [Billing alerts](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/monitor_estimated_charges_with_cloudwatch.html) for your account ([jump](#billing-alerts))
 
-The VPC template is a requirement for the others. You can either run the templates/vpc.cfn.yml template by itself prior to using the others, or run any one of the vpc-\*.cfn.yml wrapper templates at the top level of this repo to create sets of resources. For example, vpc-bastion-fargate-rds.cfn.yml will create a single stack containing a vpc, bastion host, fargate cluster, and database.
+StartupKit is design to be modular. Some stacks depend on others, some can be deployed individually or in combination with others. You can use the stacks for each module individually and combine them on your own, or use wrapper stacks we have created from the [tables below](#launch-stack) that provide one-click launch for common combinations. Finally, we've regionalized the wrapper stacks so you know which regions the stack will deploy in and which they won't.
```
Comment (rnzsgh) [inline README.md]: designed ! design

Comment (rnzsgh) [reply README.md]: "Finally, we've regionalized the wrapper stacks so you know which regions the stack will deploy in and which they won't."
How about?
The wrapper stacks in the one-click launch options are broken down by regions in order to simplify deployments. See the [service region table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) for more information on availability by region.

**Comment by rcowper** [comment]
Changes made as per @rnzsgh comments.

**Comment by rnzsgh** [comment]
Closing due to awonky squash.

**Pull request closed without merging**